Activities/Cryptography/UiPath.Cryptography.Activities/EncryptText.cs

```diff
@@ -117,8 +130,15 @@ protected override string Execute(CodeActivityContext context)
                     throw new ArgumentNullException(Resources.Activity_KeyedHashText_Property_KeySecureString_Name);
                 }
 
+                if (keyEncoding == null && string.IsNullOrEmpty(keyEncodingString)) throw new ArgumentNullException(Resources.Encoding);
+
                 if (keyEncoding == null)
-                    throw new ArgumentNullException(Resources.Encoding);
+                {
+                    //for modern and cross projects - use the string code page to get the encoding
```

Comment:
This block repeats throughout, please consolidate in a helper for consistency.

Reply:
done